docs/releasing/publishing.md

```diff
@@ -1,17 +1,35 @@
 # Publishing GOV.UK Frontend
 
-1. Make sure you have cards in the "Backlog" column to:
-  - Update the GOV.UK Design System to use the latest release
-  - Update the GOV.UK Prototype Kit to use the latest release
-  - Update the GOV.UK Frontend Docs to use the latest release
+## Steps you must complete before you release a new version of GOV.UK Frontend
 
-2. Checkout **master** and pull latest changes.
+1. Raise new issues in the team GitHub repositories to:
```

Comment:
It might be worth numbering the whole list using `1`- as far as I know, Markdown handles numbering automatically. This would make it easier to change instructions and move things around in the future.

Reply:
@EoinShaughnessy was keen to re-start the numbers from 1. for each section.

Reply:
That's interesting re Markdown — didn't know that. I think users might find a procedure of 20+ steps harder to get through without erring than a sequence of shorter procedures. Also, in a long procedure like that, it might be hard for them to relocate a particular step. To be honest, though, I don't know whether there's a GDS convention re length of procedures (nothing in the style guide anyway). Maybe 2i will shed light.